### PR TITLE
Captcha

### PR DESCRIPTION
**Pull request by filipeximenes**

### CODE REVIEW

**Comment by fjsj** [comment]
@filipeximenes build still failing

File: core/utils.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+def get_client_ip(request):
+    x_forwarded_for = request.META.get('HTTP_X_FORWARDED_FOR')
+    if x_forwarded_for:
+        ip = x_forwarded_for.split(',')[0]
```
Comment (fjsj) [inline core/utils.py]: if there is no `HTTP_X_FORWARDED_FOR`, maybe this will throw an exception

Comment (fjsj) [inline core/utils.py]: I know this is set by Heroku, but what about localhost? Or testing?

File: drafts/forms.py [hunk 2]
```diff
@@ -14,10 +17,26 @@ class Meta:
...
             for field in initial:
                 if field in self.fields:
                     self.fields[field].widget.attrs['readonly'] = True
+
+    def clean_recaptcha(self):
+        code = self.cleaned_data['recaptcha']
+        if not settings.RECAPTCHA_SECRETE_KEY:
+            ip_address = get_client_ip(self.request)
+            response = requests.post('https://www.google.com/recaptcha/api/siteverify',
+                                     data={'secret': settings.RECAPTCHA_SECRETE_KEY,
```
Comment (fjsj) [inline drafts/forms.py]: RECAPTCHA_SECRETE_KEY -> RECAPTCHA_SECRET_KEY

File: fix_my_django/settings/local.py.example [hunk 3]
```diff
@@ -32,4 +32,7 @@ REST_FRAMEWORK['DEFAULT_RENDERER_CLASSES'] = (
 )
 
 FIX_MY_DJANGO_ADMIN_MODE = True
-LOG_REQUESTS = False
+LOG_REQUESTS = False\
```
Comment (fjsj) [inline fix_my_django/settings/local.py.example]: is this `\` a typo?

File: templates/drafts/create.html [hunk 4]
```diff
@@ -56,9 +56,18 @@
...
+                </div>
+              </div>
+            </div>
           <button class="btn btn-success pull-right" type="submit">Send</button>
         </form>
       </div>
     </div>
   </div>
+
+  <script src='https://www.google.com/recaptcha/api.js'></script>
```
Comment (fjsj) [inline templates/drafts/create.html]: You can put this on `{% block js %}`

Comment (filipeximenes) [inline templates/drafts/create.html]: Does not work because this block is inside compress

File: fix_my_django/settings/local.py.example (hunk 3 above)
Comment (filipeximenes) [inline fix_my_django/settings/local.py.example]: yes, thanks

File: drafts/forms.py (hunk 2 above)
Comment (filipeximenes) [inline drafts/forms.py]: Thanks

File: core/utils.py (hunk 1 above)
Comment (filipeximenes) [inline core/utils.py]: This works locally and in tests as well

**Pull request merged**